Source/ARTRealtimePresence.m

```diff
@@ -415,7 +451,7 @@ - (BOOL)syncComplete {
 }
 
 - (BOOL)syncComplete_nosync {
-    return _channel.presenceMap.syncComplete;
+    return !(_syncState == ARTPresenceSyncInitialized || _syncState == ARTPresenceSyncStarted);
```

Comment:
should be simplified as 
```suggestion
    return _syncState == ARTPresenceSyncEnded && _syncState != ARTPresenceSyncStarted);
```

Reply:
This is as per https://github.com/ably/ably-dotnet/blob/f7b18f4902e42e90697e7fd35021025574edf650/src/IO.Ably.Shared/Realtime/Presence.cs#L44C50-L44C63 and
https://github.com/ably/ably-go/blob/dd8236fc301eb078803e83efee7615f3ba133341/ably/realtime_presence.go#L192

Reply:
I don't think this expression makes sense, because it's a single variable which if it's equal to `ARTPresenceSyncEnded` it's obviously already not `ARTPresenceSyncStarted`. For ably-go it's two different variables and in that case it does make sense to compare like that.

Reply:
Then let's make it 
```
return _syncState == ARTPresenceSyncEnded
```

Reply:
I think current expression evaluates to 
```
return _syncState == ARTPresenceSyncEnded || _syncState == ARTPresenceSyncFailed
```
We don't have `ARTPresenceSyncFailed` state in other SDKs though

Reply:
Is it ok that in this case the `ARTPresenceSyncFailed` state will be ignored? I mean `syncComplete` is true when in failed state for the old logic. It will be false after this proposal.

Reply:
https://github.com/ably/ably-cocoa/pull/1870/commits/cb04f96f05d4ac5205a7b243a96583d3a2f11f75